### PR TITLE
[B2BP-309] Configuring Strapi to return CDN URL for images src

### PR DESCRIPTION
**Pull request by simosalsi**
Depends on #131

Strapi currently inserts the AWS Media Library's URL into images src.
This PR is meant to make Strapi insert the CDN's URL instead.

#### List of Changes

- Added new properties CDN_URL and AWS_BUCKET_ENDPOINT

#### Motivation and Context

Without the CDN's URL, the static website cannot render images.

#### How Has This Been Tested?

Tested in a private environment equivalent to our production environment.

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/strapi-cms/config/plugins.ts [hunk 1]
```diff
@@ -3,14 +3,17 @@ export default ({ env }: any) => ({
...
       providerOptions: {
-        accessKeyId: env('AWS_ACCESS_KEY_ID'),
-        secretAccessKey: env('AWS_ACCESS_SECRET'),
-        region: env('AWS_REGION'),
-        params: {
-          ACL: 'private',
-          signedUrlExpires: 15 * 60,
-          Bucket: env('AWS_BUCKET_NAME'),
-        },
+        baseUrl: env('CDN_URL'),
```
Comment (datalek) [inline apps/strapi-cms/config/plugins.ts:6]: Does this configuration (`baseUrl`) work with both local providers and the S3 provider?

Comment (simosalsi) [reply apps/strapi-cms/config/plugins.ts:6]: I don't believe it does. I haven't tested it though.

It's mentioned in the [docs for the AWS provider](https://market.strapi.io/providers/@strapi-provider-upload-aws-s3) so I would assume it's meant for that.

The docs for these providers are poor at best honestly.

**Comment by maurodandrea** [comment]
@simosalsi: I created PR (https://github.com/pagopa/b2b-portals/pull/131) . If you haven't already done, you can update the code with the changes tested yesterday in our AWS environment

**Review by maurodandrea — APPROVED** [review]
@simosalsi: I created PR (https://github.com/pagopa/b2b-portals/pull/131) . If you haven't already done, you can update the code with the last changes tested yesterday in our AWS environment

**Review by datalek — APPROVED** [review]

**Pull request merged**